core/templates/dev/head/pages/exploration-editor-page/services/change-list.service.ts

```diff
@@ -28,56 +28,58 @@ angular.module('oppia').factory('ChangeListService', [
   function(
       $log, $rootScope, AlertsService, AutosaveInfoModalsService,
       ExplorationDataService) {
-    // TODO(sll): Implement undo, redo functionality. Show a message on each
-    // step saying what the step is doing.
-    // TODO(sll): Allow the user to view the list of changes made so far, as
-    // well as the list of changes in the undo stack.
+    var ChangeListService = function() {
+      // TODO(sll): Implement undo, redo functionality. Show a message on each
+      // step saying what the step is doing.
+      // TODO(sll): Allow the user to view the list of changes made so far, as
+      // well as the list of changes in the undo stack.
 
-    // Temporary buffer for changes made to the exploration.
-    var explorationChangeList = [];
-    // Stack for storing undone changes. The last element is the most recently
-    // undone change.
-    var undoneChangeStack = [];
+      // Temporary buffer for changes made to the exploration.
+      this.explorationChangeList = [];
+      // Stack for storing undone changes. The last element is the most recently
+      // undone change.
+      this.undoneChangeStack = [];
```

Comment:
Let's prefix these with `_` to signal that code should not directly change them.